dotnet-desktop-guide/net/wpf/toc.yml

```diff
@@ -47,5 +47,18 @@ items:
     items:
     - name: Overview
       href: data/index.md
+  - name: WPF resource system
+    items:
+    - name: Overview
+      href: systems/xaml-resources-overview.md
+    - name: Merged dictionaries
+      href: systems/xaml-resources-merged-dictionaries.md
+    - name: Resources in code
+      href: systems/xaml-resources-and-code.md
+    - name: Common tasks
+      items:
+      - name: Define and reference resources
```

Comment:
Are you planning to add other common tasks here? If not, I'd just move this up a level.

Reply:
Yep!